### PR TITLE
mpw 2.1-cli4 (new formula)

### PR DESCRIPTION
**Pull request by lhunath**
Master Password is a stateless password manager / generator.
It removes the dependency on backups, a vault to synchronize, and a
transport or cloud to trust.  It also makes you immune to loss.

### CODE REVIEW

File: Library/Formula/mpw.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require "formula"
+
+class Mpw < Formula
+  homepage "http://masterpasswordapp.com"
+  url "https://ssl.masterpasswordapp.com/mpw-2.1-cli3-3-gd5ff215.tar.gz"
```
Comment (mistydemeo) [inline Library/Formula/mpw.rb]: Just to check, is this a specific point release?

Comment (lhunath) [inline Library/Formula/mpw.rb]: The release is 2.1-cli3.  3 points away due some homebrew-specific build script changes.

Comment (mistydemeo) [inline Library/Formula/mpw.rb]: Perfect, thanks! So future releases should be able to use the point releases directly?

**Comment by mistydemeo** [comment]
Installation is failing on Mt. Lion: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20134/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/install_mpw/

File: Library/Formula/mpw.rb (hunk 1 above)
Comment (mistydemeo) [inline Library/Formula/mpw.rb]: Can you add an explicit `version` tag? It's being misdetected as `215` due to the unusual filename.

**Comment by lhunath** [comment]
I've pushed changes that should help get build going on other platforms.

File: Library/Formula/mpw.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  version "2.1-cli4"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "openssl"
+
+  resource "libscrypt" do
+    url "http://masterpasswordapp.com/libscrypt-b12b554.tar.gz"
+    sha1 "ee871e0f93a786c4e3622561f34565337cfdb815"
+    version "b12b554"
```
Comment (mistydemeo) [inline Library/Formula/mpw.rb]: You can leave the `version` out for the resource; it's only needed for the formula itself.

**Comment by lhunath** [comment]
done.  (Should I be commenting this or do you get a notification on a push alone?)

**Comment by mistydemeo** [comment]
I don't get notifications on pushed commits, so it's helpful - thanks!

**Pull request closed without merging**